.azure/pipelines/helix-matrix.yml

```diff
@@ -0,0 +1,63 @@
+# We only want to run full helix matrix on master
+pr: none
+trigger: none
+schedules:
+- cron: "0 */12 * * *"
+  branches:
+    include:
+    - master
+  always: true
+    
+variables:
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+  - name: _UseHelixOpenQueues
+    value: 'true'
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - group: DotNet-HelixApi-Access
+  - name: _UseHelixOpenQueues
+    value: 'false'    
+    
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Helix_matrix_x64
+    jobDisplayName: 'Tests: Helix full matrix x64'
+    agentOs: Windows
+    timeoutInMinutes: 240
+    steps:
+    # Build the shared framework
+    - script: ./build.cmd -ci -nobl -all -pack -arch x64 /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+      displayName: Build shared fx
+    - script: .\restore.cmd -ci /p:BuildInteropProjects=true
+      displayName: Restore interop projects
+    - script: .\build.cmd -ci -nobl -NoRestore -test -noBuildJava -all -projects eng\helix\helix.proj /p:IsHelixDaily=true /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildInteropProjects=true /p:RunTemplateTests=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
```

Comment:
I may have missed an earlier discussion. Why is `-noBuildJava` needed here?

Reply:
Good question, looks like we have a difference between the quarantined tests and what was in the normal ci.yml

I copied this from the quarantined test yml which had that...

https://github.com/dotnet/aspnetcore/blob/master/.azure/pipelines/quarantined-tests.yml#L38

@BrennanConroy do we need to build java for the signalr tests?

Reply:
> @BrennanConroy do we need to build java for the signalr tests?

I believe so. Also, is this pipeline running anywhere? I don't see a run of this.

Reply:
Should be here, https://dev.azure.com/dnceng/public/_build?definitionId=837&_a=summary

I'll add back the java flags on those pipelines

Reply:
Ah new pipeline name, that's why I didn't see it.